x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_customization.md

```diff
@@ -428,4 +523,142 @@ And additionally this request contains one or more custom rules
 Then the response should only list the custom rules as updated
 And all prebuilt rules should be listed as not updated
 And for each prebuilt rule the response should contain a message that the action is not allowed under current license
+```
+
+### Reverting a rule to Elastic version
+
+#### **Scenario: User can revert customized prebuilt rule to current Elastic version**
+
+**Automation**: 1 cypress test and 1 integration test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+When a user reverts the rule
+Then the rule should be modified to match the original Elastic rule object corresponding to its version number
+And the rule's `is_customized` value should be false
+```
+
+#### **Scenario: User can view diff between current and original Elastic rule versions in flyout**
+
+**Automation**: 1 cypress test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+When a user navigates to that rule's details page
+And clicks the overflow actions button
+And clicks the revert rule button
+Then a rule diff flyout should open
+And this flyout should be read-only
+And should list all fields that are different between the current and base version
+And should contain a button to revert the rule
+```
+
+#### **Scenario: Revert prebuilt rule button should be disabled when rule's base version is missing**
```

Comment:
Wouldn't it be better to hide the button when base version is missing? Otherwise it'd be nice to have an explanation tooltip why users can't revert the rule customizations.

Reply:
We do have an informational tooltip on hover, it's in the test case as the last line. We do this as it provides more information to the user as to why the feature is not available instead of just not showing the button. Didn't want to put too much information into the test case title so went with what you see here